wikia/wikia.py

```diff
@@ -395,7 +395,10 @@ def images(self):
           # Remove the /revision/ fluff after the image url
           image = image.partition("/revision/")[0]
           image_type = mimetypes.guess_type(image)[0]
-          image_type = "." + image_type.split("/")[-1]
+          if image_type is None:
+            image_type = "." + image_type.split("/")[-1]
```

Comment:
I think you messed up your conditionals here, shouldn't this be `if image_type is not None`?